packages/tonal/index.ts

```diff
@@ -18,8 +18,8 @@ import Scale from '@tonaljs/scale';
 import ScaleType from '@tonaljs/scale-type';
 import TimeSignature from '@tonaljs/time-signature';
 import Voicing from '@tonaljs/voicing';
-import VoicingDictionary from '@tonaljs/voicing-dictionary';
 import VoiceLeading from '@tonaljs/voice-leading';
+import VoicingDictionary from '@tonaljs/voicing-dictionary';
```

Comment:
hmm the pre commit hook messed something up here. I actually wanted to revert formatting everything with single quotes

Reply:
Tonal enforces [prettier](https://prettier.io/) to be used, but it seems that is not working with you? 🤔  You can `npm run format` to format the code (that's what the pre commit hook does). And it should result with double quotes. I don't know why is not the case...

Reply:
I assume it uses my local prettier config because the project does not contain a prettier config

Reply:
`npm run format` also formats everything wront (things that I did not even touch)

Reply:
it might help adding a .prettierrc with your local setup

Reply:
You're right! I thought it was one... let me add it 👍

Reply:
I've added a prettier configuration to package.json file: https://github.com/tonaljs/tonal/commit/638b5efb88c18f946cdbb30d3349db6e127069fa

Can you try to add the line `  "prettier": {},` to your package.json to see if it solves the problem?